datalad/resources/website/tests/test.html

```diff
@@ -91,6 +91,14 @@
 	  assert.deepEqual(click_handler({name: 'ds2', type: 'annex'}), {next: '?dir=/d1/ds2', type: 'search'}, 'url does not change if click current node');
 	  assert.deepEqual(click_handler({name: 'd2', type: 'dir'}), {next: '?dir=/d1/d2', type: 'search'}, 'url does not change if click current node');
       });
+      QUnit.test("Size Rendering", function(assert) {
+	assert.equal(size_renderer({ondisk: "0 Bytes", total: "1 Byte"}), "-/1 Byte", "ondisk size equals 0");
+	assert.equal(size_renderer({ondisk: "1 Byte", total: "1 Byte"}), "1 Byte", "ondisk size equals 0");
+	assert.equal(size_renderer({ondisk: "1 Byte", total: "2 Byte"}), "1 Byte/2 Byte", "ondisk size equals 0");
+	assert.equal(size_renderer({total: "1 Byte"}), "-/1 Byte", "ondisk size equals 0");
+	assert.equal(size_renderer({}), "-/-", "ondisk size equals 0");
```

Comment:
what are those "ondisk size equals 0" for all of them?  
instead of '-/-' make it just '-'? (thus uniform with above for size)

BUT!  I don't think that it should do those checks on 'human readable' strings but rather on original sizes.  E.g. if I have a directory with some small files missing (e.g. on disk 1024 bytes out of 1030 bytes total), overall total size in human readable might be the same (e.g. 1KB) but I want to have indication that I do have some files missing their load, so size indicator should be "1 KB/1KB".  for some it might be confusing -- for "inspired" it would be informative ;-)

BTW -- how difficult to add tooltips in that table?  see e.g.  http://www.w3schools.com/css/tryit.asp?filename=trycss_tooltip   and that is where we could explain while e.g. hovering over Size in the table header

1024